licenses/transifex.py

```diff
@@ -343,7 +343,13 @@ def check_for_translation_updates_with_repo_and_legalcodes(
                 resource_slug not in resource_slugs_on_transifex
                 or language_code not in self.stats[resource_slug]
             ):
-                raise Exception(f"Transifex has no translation for {resource_slug}")
+                logger.error(
+                    f"Transifex has no translation for {resource_slug}"
+                )  # pragma: no cover
+                print(
+                    f"ERROR: Transifex has no translation for {resource_slug}"
+                )  # pragma: no cover
```

Comment:
This change was made because we don't want to stop execution?

Reply:
Right, we still want to check the status of other things even if this one fails.

The one thing we might want to do, and don't right now, is finish the command with an error status exit code.